server/src/logic.js

```diff
@@ -210,6 +219,23 @@ export const scheduleHandler = {
   },
 };
 
+export const eventHandler = {
+  group(eventt) {
+    return eventt.getGroup();
+  },
+  responses(event) {
+    return EventResponse.findAll({
+      where: { EventId: event.id },
```

Comment:
i would normally expect `EventId` to be `eventId`

Reply:
wth, why did that even work.